app/controllers/admin/settings_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Admin
+  class SettingsController < ApplicationController
+    def index; end
+
+    def create
+      setting_params.keys.each do |key|
+        Setting.send("#{key}=", setting_params[key].strip) unless setting_params[key].nil?
+      end
+
+      flash[:success] = I18n.t('settings.success')
+      render :index
+    end
+
+    private
+
+    def setting_params
```

Comment:
Why do you use the singular `setting` here? My understanding ist that a setting is an individual configuration option, e.g. `project_name`.

Reply:
No special reason except that most people name those things after the singular of the activerecord model it's supposed to change. So, `user_params`, `post_params`, not `users_params`. Feel free to change that! That shouldn't have any side-effects.